<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="CTC_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>CTC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="track_controller_A_button">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Track Controller A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="track_controller_b_button">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>130</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Track Controller B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="track_controller_c_button">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>210</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Track Controller C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="track_model_button">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>70</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Track Model</string>
    </property>
   </widget>
   <widget class="QPushButton" name="train_model_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Train Model</string>
    </property>
   </widget>
   <widget class="QPushButton" name="train_controller_button">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>230</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Train Controller</string>
    </property>
   </widget>
   <widget class="QPushButton" name="time_module_button">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>270</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Time Module</string>
    </property>
   </widget>
   <widget class="QPushButton" name="track_controller_A_testbench_button">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Track Controller A Testbench</string>
    </property>
   </widget>
   <widget class="QPushButton" name="track_controller_B_testbench_button">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Track Controller B Testbench</string>
    </property>
   </widget>
   <widget class="QPushButton" name="track_controller_C_testbench_button">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>190</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Track Controller C Testbench</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>859</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
